src/gui/dialog/SettingsDialog.h

```diff
@@ -35,9 +35,15 @@ class SettingsDialog: public GladeGui {
     void setDpi(int dpi);
 
     /**
-     * Autosave was toggled, enable / disable autosave config
+     * Set active regions
      */
     void enableWithCheckbox(const string& checkbox, const string& widget);
+    void disableWithCheckbox(const string& checkbox, const string& widget);
+
+    /**
```

Comment:
**Nitpick (non-blocking):** I would not use `/**`, since this marks the comment as documentation for the next function and will be associated with it. Since the content is a description for the whole block, we do not want this.

```suggestion
    /*
```